### ISSUE TITLE
Multiple feeds on home page

### ISSUE THREAD

**Issue by isingh** [PeaceGeeksSociety/amani#1090]
We want to display multiple feeds on the home page

1. Ability to show most recent articles, resources and events etc.
2. Ability to add a new recent items feed without `superadmin` access

**Comment by PeaceGeeks** [comment]
isn't this the same as #789 ?

**Comment by PeaceGeeksProjects** [comment]
and #1091?

**Comment by isingh** [comment]
From what I understand #789 is about showing multiple content types inside the same block (or being able to configure). This is to have multiple such blocks. I am not too familiar with amani yet, but it seems we can add more blocks. If this functionality works, that is great, I can test it, otherwise we want to multiple front page blocks each showing a different feed of recent content.

**Comment by erica-scott** [comment]
For this issue, I built four view displays for Articles, Events, Resources, and Blog Entries. 

And then I added them to the content section of the front_page context. 

Then, the only problem was that the CSS for these blocks was specified in the way of referencing the id of the block (ex. #block-views-news-and-events-block-1) so it was necessary to add the other three blocks in to the CSS as well. The only problem with this, is that adding more feed types, would require doing this as well. 

Finally, I added some configuration settings so the Amani Config user can easily enable/disable these view blocks on the front page. 

A pull request has been made and merged into Amani-Zen-Theme. The code will need to be pushed to dev and the features reverted.
